src/commands/deploy/promptToSaveDeployDefaults.ts

```diff
@@ -15,7 +15,7 @@ export async function promptToSaveDeployDefaults(context: IActionContext, node:
     const defaultWebAppToDeploySetting: string | undefined = getWorkspaceSetting(constants.configurationSettings.defaultWebAppToDeploy, workspacePath);
     // only prompt if setting is unset
     if (!defaultWebAppToDeploySetting) {
-        const saveDeploymentConfig: string = localize('showDeploymentConfig', 'Always deploy the workspace "{0}" to "{1}"?', path.relative(workspacePath, deployPath) || path.basename(workspacePath), node.root.client.fullName);
+        const saveDeploymentConfig: string = localize('showDeploymentConfig', 'Always deploy the workspace "{0}" to "{1}"?', path.relative(workspacePath, deployPath) ? deployPath : path.basename(workspacePath), node.root.client.fullName);
```

Comment:
With the update, the prompt will display full path for nested folders and just the workspace name in the other case.